tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -107,10 +108,12 @@ Object.Dai {
 	SSP."1" {
 		id 		$SPK_ID
 		dai_index	1
-		direction	"playback"
+		direction	"duplex"
 		name		SSP1-Codec
 		default_hw_conf_id	0
 		sample_bits		32
+		# used for ref cap
```

Comment:
can we expand the comment to say reference capture please?

Reply:
Ack